src/structures/Guild.js

```diff
@@ -751,14 +751,12 @@ class Guild extends Base {
     if (this.members.has(user)) return Promise.resolve(this.members.get(user));
     options.access_token = options.accessToken;
     if (options.roles) {
-      const roles = [];
```

Comment:
This looks like an actual bug to me.
Can you readd those lines and add a `options.roles = roles;` after the loop?

Reply:
Fixed in 45ab03d